src/main/java/org/launchcode/techjobs/oo/Job.java

```diff
@@ -2,24 +2,132 @@
 
 import java.util.Objects;
 
-public class Job {
+//    private int id;
+//    private static int nextId = 1;
+//
+//    private String name;
+//    private Employer employer;
+//    private Location location;
+//    private PositionType positionType;
+//    private CoreCompetency coreCompetency;
+
+    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
+    //  other five fields. The second constructor should also call the first in order to initialize
+    //  the 'id' field.
+
+    // TODO: Add custom equals and hashCode methods. Consider two Job objects "equal" when their id fields
+    //  match.
+
+
+    // TODO: Add getters for each field EXCEPT nextId. Add setters for each field EXCEPT nextID
+    //  and id.
+import java.util.Objects;
 
+public class Job {
+    //fields
     private int id;
     private static int nextId = 1;
+    private static final String DEFAULT_MESSAGE = "Data not available";
 
     private String name;
     private Employer employer;
     private Location location;
     private PositionType positionType;
     private CoreCompetency coreCompetency;
 
-    // TODO: Add two constructors - one to initialize a unique ID and a second to initialize the
-    //  other five fields. The second constructor should also call the first in order to initialize
-    //  the 'id' field.
+    //getters
```

Comment:
I would recommend moving your constructors here to place underneath its associated comment, but also to follow a pattern that we've seen in class (i.e. Class fields - 1st, Constructor(s) - 2nd, Getters and Setters - 3rd or 4th, Methods - 3rd or 4th).